frontend/src/modules/Worksheets/services/analyzeTextDocument.js

```diff
@@ -0,0 +1,34 @@
+import { gql } from 'apollo-boost';
+
+export const analyzeTextDocument = ({
+  prompt,
+  environmentUri,
+  worksheetUri,
+  datasetUri,
+  key
+}) => ({
+  variables: {
+    prompt,
+    environmentUri,
+    worksheetUri,
+    datasetUri: datasetUri,
+    key: key
```

Comment:
why do we have a mix of kwargs and args here?

Reply:
making all positional - as we tend to do with other GQL Ops